Add-ons/UmbracoCourier/CommonIssues.md

```diff
@@ -6,21 +6,18 @@ _Document to outline commonly found issues with Courier and possible solutions_
 For all issues, always try these steps first:
 - Enable debugging in /config/courier.config and restart the app to see if courier has problems with loading dlls
 - Clear the cache folder in /app_data/courier/cache
-- Ensure you have the latest version, current 2.7.8.14
+- Ensure you have the latest version, current **3.1.7**
```

Comment:
We shouldn't mention the exact current version number, because then we'd have to go an update it in here, everytime we upgrade 😅 
My suggestion would be:
```suggestion
- Ensure you have the latest version**
```